profiles/source_profile.go

```diff
@@ -290,6 +318,14 @@ func (src SourceProfile) ToLegacyDriver(source string) (string, error) {
 		}
 	case SourceProfileTypeConfig:
 		return "", fmt.Errorf("specifying source-profile using config not implemented")
+	case SourceProfileTypeCsv:
+		{
+			if strings.ToLower(source) == constants.CSV {
+				return constants.CSV, nil
+			} else {
+				return "", fmt.Errorf("found manifest in the source profile but invalid -source flag received = %v, (did you mean csv?)", source)
```

Comment:
when will this case be encountered ? The sourceProfileType csv will only be encountered with the source is set to CSV, right ?

Reply:
Good catch, I missed updating this. This was required earlier as per the code changes but not needed now.